midterm-2023-11-07/code/5311-kristijan-lazarov/script.ts

```diff
@@ -0,0 +1,322 @@
+interface Movie {
+    id: number
+    title: string
+    year: number
+    director: string
+    genre: string[]
+    plot: string
+    cast: Actor[]
+    oscars: any
```

Comment:
The data is in a format that is common in JavaScript, so it needs to be typed better than any (actually it should be a `Record<string, string>`)